tool/fetch-dart-sdk-sums.sh

```diff
@@ -28,12 +29,15 @@ done
 BASEURL="https://storage.googleapis.com/dart-archive/channels"
 CHANNELS="stable beta dev"
 ARCHS="amd64 arm64"
+ENDING='\\\n'
+
+printf "\n$(blue "Copy the following output and replace the existing code in the Dockerfile")\n"
+printf "$(blue "inside the 'set -eu' run statement:")\n\n"
```

Comment:
**issue:** These colors don't work on the macOS. All they return is the color codes, which do nothing here. Can we find another solution?

Reply:
Also, see the comments in [this article](https://dev.to/ifenna__/adding-colors-to-bash-scripts-48g4).

Reply:
I wanted to avoid specifying the colors again as then we would be repeating functionality across the scripts and would have to remember to include `ENDCOLOR` on each one. 

The color commands were being used incorrectly in the script before, but I thought these changes fixed that. This PR works for me in the Docker container, macOS terminal with ZSH and bash shell, as well as in IntelliJ's terminal emulator.

<img width="600" alt="Terminal output" src="https://user-images.githubusercontent.com/18372958/208058373-6f977e02-2a27-4e52-b204-b324f18c11e6.png">

Could you show me the output you get with these new changes?